### ISSUE TITLE
Can't slide TX fee 

### ISSUE THREAD

**Issue by JeremiGendron** [ArkEcosystem/desktop-wallet#856]
**Platform (please complete the following information):**
 - OS: Lubuntu
- Version 18.10

**Describe the bug**
The slider is unresponsive. Console shows an error.

**How to reproduce the bug**
Steps to reproduce the behaviour:
1. Go to send a transaction
2. Click on the slider
3. Drag sideways
4. Doesn't move

**Expected behaviour**
Being able to slide the fee up and down

**Screenshots**
![image](https://user-images.githubusercontent.com/23587429/50387830-ddc43400-06d3-11e9-8f51-4667b704f0f0.png)
![image](https://user-images.githubusercontent.com/23587429/50387845-19f79480-06d4-11e9-8ade-1881aaca7e0b.png)

**Additional context**
Only tested on transfer at this point.

Investigating now

**Comment by JeremiGendron** [comment]
The other transaction types seem to work well.

**Comment by dated** [comment]
Can't reproduce on macOS.

**Comment by JeremiGendron** [comment]
Alright, I think it might be an issue with my profile configuration. This was on an account with balance, registered as delegate on devnet. I will try to reset data and see what happens.

**Comment by JeremiGendron** [comment]
Yes, the message also says `the minimum fee is DARK 1.0` OR `the maximum fee is DARK 0.10` (if the slider is at max, it says minimum, and the opposite for minimum.

**Comment by JeremiGendron** [comment]
When at the minimum location (can only get there by clicking on `Minimum` button, it days that:
![image](https://user-images.githubusercontent.com/23587429/50388341-3948ee80-06e1-11e9-94ba-efb86fb82e61.png)

When clicking on any button but the `Advanced`, it says that:
![image](https://user-images.githubusercontent.com/23587429/50388345-52ea3600-06e1-11e9-8adb-20fd8eec9ac0.png)

If clicked on `Advanced`, it slides normally
![image](https://user-images.githubusercontent.com/23587429/50388347-65646f80-06e1-11e9-8e71-f30d6967e64e.png)

**Comment by zillionn** [comment]
What's your peer IP when you experience this issue? Does changing peer resolves it?

**Comment by JeremiGendron** [comment]
All the 167.* won't work.

**Comment by zillionn** [comment]
Does changing the peer resolves your problem?

**Comment by JeremiGendron** [comment]
not at all from what I can tell, neither does changing npm version, yarn cache clean, git clone new repo, reset data.

**Comment by zillionn** [comment]
Oh, you're using the development version? Do you experience the same issue with the production? https://github.com/ArkEcosystem/desktop-wallet/releases

**Comment by JeremiGendron** [comment]
I've been using the development branch for the whole week, only had issues after rebasing yesterday at around 2pm EST IIRC. Release works, of course.
